ta/asym_cipher_perf/include/user_ta_header_defines.h

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2015, Linaro Limited
+ * All rights reserved.
+ */
+
+#ifndef USER_TA_ASYM_HEADER_DEFINES_H
+#define USER_TA_ASYM_HEADER_DEFINES_H
+
+#include "ta_asym_cipher_perf.h"
+
+#define TA_UUID TA_ASYM_CIPHER_PERF_UUID
+
+#define TA_FLAGS                (TA_FLAG_USER_MODE | TA_FLAG_EXEC_DDR | \

```

Comment:
`TA_FLAG_USER_MODE` and `TA_FLAG_EXEC_DDR` are deprecated and not needed any longer.
I'm not sure `TA_FLAG_SECURE_DATA_PATH` or `TA_FLAG_CACHE_MAINTENANCE` are needed.
We should be able to define `TA_FLAGS` to `0`.